samples/sensor/bme280/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _bme280:
+
+BME280 Humidity and Pressure Sensor
+###################################
+
+Overview
+********
+
+This sample application periodically reads temperature, pressure and humidity data from
+the first available device that implements SENSOR_CHAN_AMBIENT_TEMP, SENSOR_CHAN_PRESS
+and SENSOR_CHAN_HUMIDITY. This sample checks the sensor in polling mode (without
+interrupt trigger).
+
+Building and Running
+********************
+
+This sample application uses an BME280 sensor connected to an Arduino 101 board via I2C.
+Sensor has multiple pins so you need to connect according to connection diagram given in
+`bme280 datasheet`_ at page 38.
+There are two processor cores (x86 and ARC) on the Arduino 101. You will need to
+flash both this sample's code on the ARC core (using the ``arduino101_ss`` board target),
+and stub code on the x86 core (using the ``arduino_101`` board target), as shown below.
+
+.. code-block:: console
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensors/bme280
+   :board: arduino_101_sss
+   :goals: flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: tests/booting/stub
+   :board: arduino_101
+   :goals: flash
+   :compact:
+
+Sample Output
+=============
+To check output of this sample , any serial console program can be used.
+Here I am using picocom program to open output. Check which tty device it is.
+In my case it is ttyUSB0
+
+.. code-block:: console
+
+        $ sudo picocom -D /dev/ttyUSB0
+
+.. code-block:: console
+
+        temp: 27.950000; press: 100.571027; humidity: 61.014648
+        temp: 27.940000; press: 100.570269; humidity: 61.012695
+        temp: 27.950000; press: 100.570695; humidity: 61.002929
+
+.. _bme280 datasheet: https://www.mouser.com/ds/2/621/BST-BME280_DS001-09-521021.pdf
```

Comment:
Please link directly to the vendor: https://ae-bst.resource.bosch.com/media/_tech/media/datasheets/BST-BME280-DS002.pdf

Reply:
I thought you fixed this earlier, but it seems to have been reverted. The dts overlay disappeared too.

Reply:
I am very sorry about this. I try to works on different PC but looks it not a good idea for me now. I have correct those issues.